sklearn/feature_selection/_sequential.py

```diff
@@ -201,11 +202,23 @@ def fit(self, X, y=None):
             Target values. This parameter may be ignored for
             unsupervised learning.
 
+        **params : dict, default=None
+            Parameters to be passed to the cross_val_score function.
```

Comment:
```suggestion
            Parameters to be passed to the scoring function evaluated through cross validation 
            via the `cross_val_score` function.
```

Reply:
Within the `cross_val_score` function I think we route to more than just the scoring function. We also route to cv and estimator `fit` methods.